README.md

```diff
@@ -1,73 +0,0 @@
-# Shareaholic for Drupal Module
```

Comment:
@lukasz-zaroda We should keep a README. It's a public repo.

Reply:
I added an empty README.md although I think it would be better if a native speaker would write the content :) .

Reply:
Done